Task_TheGridProblem/index.css

```diff
@@ -1 +1,63 @@
-/* Start coding below for GRID PROBLEM TASK */
+*{
+    margin: 0px;
+    padding: 0px;
+}
+.container{
+    height: 100vh;
+    width: 100vw;
+    background: black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: "Inter", sans-serif;
+}
+.big-box{
+    background: #D9D9D9;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: 1fr 1.2fr;
+    width: 500px;
+    height: 400px;
+    gap: 3px;
+    font-size: 44px;
+    font-weight: 400;
+    text-align: left;
+    color: #FFFFFF;

```

Comment:
Unnecessary property 'padding: 0px;' since the default value is 0, can be removed